Doc/whatsnew/3.15.rst

```diff
@@ -274,6 +274,14 @@ typing
   (Contributed by Bénédikt Tran in :gh:`133823`.)
 
 
+unittest
+--------
+* Expose log formatter to users in TestCase.assertLogs.
```

Comment:
Add an empty line above.

I think that "Expose log formatter to users" does not correctly describes the feature. "Allows to specify formatter" looks more accurate.